asllib/doc/Types.tex

```diff
@@ -220,19 +230,17 @@ \subsubsection{Formally}
 \begin{mathpar}
 \inferrule[not\_well\_constrained]{
   \tty \eqname \TInt(\vc)\\
-  \astlabel(\vc) \neq \wellconstrained
+  \astlabel(\vc) \notin \{\wellconstrained,\pendingconstrained\}
 }{
   \annotatetype{\overname{\Ignore}{\vdecl}, \tenv, \tty} \typearrow \overname{\tty}{\newty}
 }
 \end{mathpar}
 
 \begin{mathpar}
-\inferrule[well\_constrained\_empty]{
-  \constraints \eqname \emptylist
-}{
+\inferrule[pending\_constrained]{}{
   {
     \begin{array}{r}
-  \annotatetype{\overname{\Ignore}{\vdecl}, \tenv, \overname{\TInt(\wellconstrained(\emptylist))}{\tty}} \typearrow
+  \annotatetype{\overname{\Ignore}{\vdecl}, \tenv, \overname{\TInt(\pendingconstrained)}{\tty}} \typearrow
```

Comment:
Please specify the type error: switch `\TypeErrorConfig` with something like `\TypeErrorVal{NonPendingTypeExpected}`